Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin-release:golang-1.10 as builder
+FROM openshift/origin-release:golang-1.12 as builder
```

Comment:
What level of testing do we need to do in order to ensure a change to the go version is okay?

Reply:
Good question. For this PR I will verify all functionality of the MSS after updating to this image so as to ensure no broken functionality.